### PR TITLE
deprecated erlang:get_stacktrace/0 function

### PR DESCRIPTION
**Pull request by getong**

### CODE REVIEW

**Comment by eproxus** [comment]
Hi @getong, thanks for your PR! Would you mind using the same macros as in https://github.com/eproxus/meck/pull/193? That way, the arrow in the try-catch statement stays intact and the source looks more like proper Erlang.

**Comment by getong** [comment]
@eproxus The macro is referenced from [rebar3](https://github.com/erlang/rebar3), and I think this works fine, you can take a look at it.
https://github.com/erlang/rebar3/blob/master/src/rebar.hrl

**Comment by eproxus** [comment]
My issue is not that it doesn't work, it's that it breaks analysis tools and the flow of reading the code. We would like [this macro](https://github.com/eproxus/meck/blob/master/src/meck_code_gen.erl#L186-L192) and [this usage](https://github.com/eproxus/meck/blob/master/src/meck_code_gen.erl#L203-L205) for this PR.

File: src/grisp_onewire.erl [hunk 1]
```diff
@@ -131,8 +132,8 @@ handle_call({transaction, Fun}, _From, State) ->
     Reply = try
         {result, Fun()} % TODO: Implement timeout for transactions
     catch
-        Class:Reason ->
-            {exception, Class, Reason, erlang:get_stacktrace()}
+	?EXCEPTION(Class, Reason, StackToken) ->
```
Comment (eproxus) [inline src/grisp_onewire.erl]: Looks good! Could you please update the variable name to `Stacktrace` here and below?

**Pull request merged**